Makefile

```diff
@@ -79,23 +91,42 @@ devenv: $(wildcard $(subst dir, devenv, $(TWO_DIRS_DEEP)))
 	docker compose -f docker-compose.emily.yml build
 	@touch devenv
 
-ifneq ($(filter arm64 aarch64, $(shell uname -m)),)
-_LAMBDA_FLAGS := --arm64
-endif
+# Emily CDK Template ---------------------------------
+
+EMILY_CDK_SOURCE_FILES :=
+EMILY_CDK_SOURCE_FILES := $(wildcard $(subst dir, emily/cdk/lib, $(FIVE_DIRS_DEEP))) $(EMILY_CDK_SOURCE_FILES)
+EMILY_CDK_SOURCE_FILES := $(wildcard $(subst dir, emily/bin/lib, $(FIVE_DIRS_DEEP))) $(EMILY_CDK_SOURCE_FILES)
```

Comment:
Why not change this to the following:
```makefile
EMILY_CDK_SOURCE_FILES := $(wildcard $(subst dir, emily/cdk/lib, $(FIVE_DIRS_DEEP)))
EMILY_CDK_SOURCE_FILES := $(wildcard $(subst dir, emily/bin/lib, $(FIVE_DIRS_DEEP))) $(EMILY_CDK_SOURCE_FILES)
```

This is more educational for me than anything, I don't know that much about what is or isn't legal in a `Makefile`.

Reply:
It just seemed more idiomatic to me to be honest, but I might be inventing my own standard. Originally I had this sequence in front of a number of `Make` commands and it looked like this:

```Make
SOURCE_FILES :=
SOURCE_FILES := $(wildcard $(subst dir, emily/cdk/lib, $(FIVE_DIRS_DEEP))) $(SOURCE_FILES)
SOURCE_FILES := $(wildcard $(subst dir, emily/bin/lib, $(FIVE_DIRS_DEEP))) $(SOURCE_FILES)
```

And then I'd have that in front of every make command so that the longer make commands all followed the same structure. What do you think @djordon? Otherwise I can change these lines to be what you're suggesting.

Reply:
Ahh I see. Well, they seem equivalent and equally clear so I have a slight preference for the shorter version, but if one is more idiomatic (even if it's your own standard) then we can go with that.

Reply:
I changed it to your suggestion, no need to make up idioms :)